arch/x86/adler32_avx2.c

```diff
@@ -1,17 +1,154 @@
 /* adler32_avx2.c -- compute the Adler-32 checksum of a data stream
- * Copyright (C) 1995-2011 Mark Adler
+ * Copyright (C) 1995-2022 Mark Adler
```

Comment:
This change seems bad... Copyright doesn't automagically extend unless original author makes further changes and they are merged in. However Mark Adler isn't mentioned in the list of contributing authors.

Reply:
Ah, fixing now. He isn't an author, except for the core algorithm, anyway.  That was already there originally when I had gotten to it (the 1995-2011 bit).